### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/react_goods-selector-hooks/)

### CODE REVIEW

**Review by RomeoKs — APPROVED** [review]
Nicely done.
Learned a few thing from you code actually.

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +17,94 @@ export const goods = [
...
+
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+
+  const clearSelected = () => setSelectedGood('');
+  const isSelected = (good: string) => selectedGood === good;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const getButtonProps = (good: string): Button => ({
```
Comment (naziamoff) [inline src/App.tsx:33]: wow, metabutton 😄

**Review by naziamoff — APPROVED** [review]

**Comment by lefteris-L** [comment]
> Nicely done. Learned a few thing from you code actually.

Thanks man, that's awesome. You should review more often

File: src/App.tsx (hunk 1 above)
Comment (lefteris-L) [reply src/App.tsx:33]: I had to google that, but yeah exactly 😅

**Pull request closed without merging**